std_msgs/Header header
float64 mse
float64 instant_error_x
float64 instant_error_y
float64 instant_error_theta
float64 total_xy
